### ISSUE TITLE
[AGWG] PR request for ACT Rules Format 1.0

### ISSUE THREAD

**Issue by nitedog** [w3c/transitions#151]
# Document title, URLs, estimated publication date

Title: Accessibility Conformance Testing (ACT) Rules Format 1.0

Staging draft ready for publication (meets PubRules): https://www.w3.org/2017/09/ACT/WD-act-rules-format-10-20170912/ 

Planned Publication URI: https://www.w3.org/TR/2019/WD-act-rules-format-1.0-20190730/ 
Latest version URI: https://www.w3.org/TR/act-rules-format/ 

Expected publication date: 30 July 2019

# Abstract

The Accessibility Conformance Testing (ACT) Rules Format 1.0 defines a format for writing accessibility test rules. These test rules can be used for developing automated testing tools and manual testing methodologies. It provides a common format that allows any party involved in accessibility testing to document and share their testing procedures in a robust and understandable manner. This enables transparency and harmonization of testing methods, including methods implemented by accessibility test tools.

# Status

This section describes the status of this document at the time of its publication. Other documents may supersede this document. A list of current W3C publications and the latest revision of this technical report can be found in the W3C technical reports index at https://www.w3.org/TR/. 

This document is a Proposed Recommendation of Accessibility Conformance Testing Rules Format 1.0 (ACT Rules Format 1.0), published by the Accessibility Guidelines Working Group. ACT Rules Format 1.0 was published as a Candidate Recommendation on 16 April 2019. Since then the Working Group has collected implementations in the form of conforming rules. No changes were made to normative requirements clauses. A history of changes to ACT Rules Format 1.0 is available in the appendix.

The primary purpose of the Proposed Recommendation is to collect feedback from the Advisory Committee. W3C Advisory Committee Members are invited to send formal review comments to the W3C Team until 24 September 2019. Comments should be made using the Call for Review WBS form. Advisory Committee Representatives should consult their WBS questionnaires.

The W3C Membership and other interested parties are invited to review the document and send comments. Note that substantive technical comments were expected during the Candidate Recommendation review period that ended 28 May 2019. To comment, aside from Advisory Committee comments, file an issue in the W3C ACT Rules Format 1.0 GitHub repository. The Working Group requests that public comments be filed as new issues, one issue per discrete comment. It is free to create a GitHub account to file issues. If filing issues in GitHub is not feasible, send email to public-wcag-act-comments@w3.org (comment archive). Comments are requested by 24 September 2019. In-progress updates to the document may be viewed in the publicly visible editors' draft.

Please see the Working Group's implementation report. 

Publication as a Proposed Recommendation does not imply endorsement by the W3C Membership. This is a draft document and may be updated, replaced or obsoleted by other documents at any time. It is inappropriate to cite this document as other than work in progress. 

This document was produced by a group operating under the W3C Patent Policy. W3C maintains a public list of any patent disclosures made in connection with the deliverables of the group; that page also includes instructions for disclosing a patent. An individual who has actual knowledge of a patent which the individual believes contains Essential Claim(s) must disclose the information in accordance with section 6 of the W3C Patent Policy.
This document is governed by the 1 March 2019 W3C Process Document.

# Link to group's decision to request transition

https://lists.w3.org/Archives/Public/w3c-wai-gl/2019JulSep/0090.html

# Changes

Change history: https://w3c.github.io/wcag-act/act-rules-format.html#Change_History

There have been no substantive changes since the prior Candidate Recommendation Working Draft.

Changes were all editorial and relatively minor. In a few places, text was re-arranged to improve clarity and flow (eg. text on versioning in section 4.11. This re-ordering does not impact requirements clauses nor change the overall meaning. None of the other changes impacted the requirements clauses either.

# Requirements satisfied

The group believes that the document meets the requirements initially established:
https://w3c.github.io/wcag-act/act-fr-reqs.html

# Dependencies met (or not)

This document does not have dependencies, except RFC 2119 to define keywords. Even though this document was primarily developed for WCAG conformance testing, it is intentionally written to be agnostic to WCAG and any other accessibility standard. For example, this document can be used to create test rules for WAI-ARIA conformance. There are informative references to these standards.

# Wide Review

A Wide Review Working Draft was published on 5 July 2018. No comments were received that required substantive changes to the Candidate Recommendation Working Draft published on 16 April 2019. The group followed the checklists for Accessibility, Internationalization, and Privacy/Security, and did not identify any issues relating to these aspects because this document does not specify a new web technology.

# Issues addressed

The group believe to have addressed all comments received on the previous draft:
https://github.com/w3c/wcag-act/labels/CR%20feedback

# Formal Objections

There were no objections raised for this publication.

# Implementation

An implementation report following the test plan established for the Candidate Recommendation Working Draft documents rules that meet the features specified by the document:
https://w3c.github.io/wcag-act/act-implementations.html

This selection of rules is part of a larger set of rules being continually developed by the ACT Rules Community Group (previously called auto-WCAG CG):
https://act-rules.github.io/rules/

The implementation report also documents rules that are already implemented by a variety of vendors in automated testing tools and in manual testing methodologies. This is attracting additional vendors who are in the process of implementing rules as well. The group believes this meets the requirements for demonstrating interoperability and implementation of the ensuing standard.

# Patent disclosures

No patent disclosures have been made to date. Patent disclosures for this group are tracked at:
https://www.w3.org/2004/01/pp-impl/35422/status

**Comment by swickr** [comment]
Transition approved.

Thanks for all this work.

**Comment by plehegar** [comment]
Published https://www.w3.org/TR/2019/PR-act-rules-format-1.0-20190730/
